Formula/groovyserv.rb

```diff
@@ -14,17 +14,13 @@ class Groovyserv < Formula
   end
 
   depends_on "go" => :build
-
-  # This fix is upstream and can be removed in the next released version.
-  patch do
-    url "https://github.com/kobo/groovyserv/commit/4ea88fbfe940b50801be5e0b0bc84cd0ce627530.diff"
-    sha256 "b7de43a030f97d8368ad9a9b895bd1242abc53c4ce0f12e7ab2acfc82e97da65"
-  end
+  depends_on "gradle" => :build
+  depends_on "groovy" => :run
```

Comment:
Nout? It's British slang 🙈. Just means "nothing".

"How much of the money James owes did he give back?"
"Nout 😒"

etc.